.gitattributes

```diff
@@ -4,5 +4,21 @@
 *.png binary
 *.jpg binary
 
-#Files we do not want to distribute
-/tests export-ignore
\ No newline at end of file
+# Directories and files we do not want to distribute
+/.build/              export-ignore
+/.ci/                 export-ignore
+/.github/             export-ignore
+/devTools/            export-ignore
+/tests/               export-ignore
+/.editorconfig        export-ignore
+/.gitattributes       export-ignore
+/.github              export-ignore
```

Comment:
This is probably not needed, can we remove it? We already have a rule above for `/.github/`.

Reply:
😃 you are correct.

done.